metrics/http_middleware.lua

```diff
@@ -71,6 +71,9 @@ end
 -- ... arguments for pcall to instrument
 function export.observe(collector, route, ...)
     return collector:observe_latency(function(ok, result)
+        if result == nil then
+            error("incorrect http handler", 0)
```

Comment:
I think it should be more explicit, like `expecting {status = status, ...}` return

Reply:
But status can be omitted: 

```lua
status = (not ok and 500) or result.status or 200,
```